package.json

```diff
@@ -99,6 +99,7 @@
   "devDependencies": {
     "@babel/cli": "7.0.0-rc.1",
     "@babel/core": "7.0.0-rc.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-rc.1",
```

Comment:
Hmm, maybe we should not have used `^` here like the rest of the non-stable deps.